src/lib/models/fb.ts

```diff
@@ -335,10 +333,8 @@ export default class Facebook {
       );
       if (postHnd?.toString() !== 'JSHandle:undefined') {
         const postData = await this.parsePost(<ElementHandle>postHnd);
-        if (callback !== undefined && callback !== null) {
-          callback(postData);
-        }
-        savePost(postData);
+        if (callback) callback(postData);
+        if (save) savePost(postData);
```

Comment:
@kaanyagci I added that check here.

Reply:
Yes in git diff I was unable to see the function name but actually you're right.